README.md

```diff
@@ -119,6 +119,7 @@ Table of Contents
   * http://vaddy.net/ - Continuous web security testing with continuous integration (CI) tools. 3 domains, 10 scan history for free
   * https://www.globalsign.com/en/ssl/ssl-open-source/ - Free SSL certs for Open Source projects
   * https://www.startssl.com/ - Free SSL certs
+  * https://buy.wosign.com/free/ - Free SSL certs. Up to 100 subdomains. China authority
```

Comment:
the cert on their own site is a SHA-1 cert and the CA cert is also sha-1, they seem to suggest they can issue sha-2 certs for the free ones but meh, rather not support people who still use sha-1 anyway